01-CSS_Transform/assignment/css/style.css

```diff
@@ -0,0 +1,202 @@
+  border-right: ($menu_WIDTH / 2) solid transparent;            //using variables
+  border-bottom: 0 solid #ebebeb;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  z-index: 101;
+  transition: 0.2s border-bottom ease-in;
+}
+
+.menu:hover > .menu-title:after {
+  border-bottom-width: 5px;
+  transition: 0.2s 0.2s border-bottom-width ease-out;
+}
+
+.menu-title_2nd { background: #DC143C; // background of title2 }
+.menu-title_2nd:hover { background: #3e86a3; }
+.menu-title_2nd:before { border-top-color: #4e96b3; }
+.menu-title_2nd:hover:before { border-top-color: #3e86a3; }
+
+.menu-title_3rd { background: #00CED1; // background of title3 }
+.menu-title_3rd:hover { background: #b96666; }
+.menu-title_3rd:before { border-top-color: #c97676; }
+.menu-title_3rd:hover:before { border-top-color: #b96666; }
+
+.menu-title_4th { background: #E9967A; // background of title4 }
+.menu-title_4th:hover { background: #cb9b48; }
+.menu-title_4th:before { border-top-color: #dbab58; }
+.menu-title_4th:hover:before { border-top-color: #cb9b48; }
+
+
+.menu-dropdown {      //dropdown menu
+  min-width: 100%;
+  padding: 15px 0;
+  position: absolute;
+  background: #ebebeb;
+  z-index: 100;
+  transition:
+    0.5s padding,
+    0.5s background;
+}
+
+.menu-dropdown:after {
+  content: "";
+  display: block;
+  height: 0;
+  border-top: 5px solid #ebebeb;
+  border-left: ($menu_WIDTH / 2) solid transparent;               //using variables
+  border-right: ($menu_WIDTH / 2) solid transparent;              //using variables
+  position: absolute;
+  top: 100%;
+  left: 0;
+  z-index: 101;
+  transition: 0.5s border-top;
+}
+
+.menu:not(:hover) > .menu-dropdown {
+  padding: 4px 0;
+  background: #dddddd;
+  z-index: 99;
+}
+
+.menu:not(:hover) > .menu-dropdown:after {
+  border-top-color: #dddddd;
+}
+
+.menu:not(:hover) > .menu-title:after {
+  border-bottom-color: #dddddd;  //drop-down menu cover
+}
+
+.menu-dropdown > * {
+  overflow: hidden;
+  height: 30px;
+  padding: 5px 10px;
+  background: rgba(0,0,0,0);
+  white-space: nowrap;
+  transition:
+    0.5s height cubic-bezier(.73,.32,.34,1.5),
+    0.5s padding cubic-bezier(.73,.32,.34,1.5),
+    0.5s margin cubic-bezier(.73,.32,.34,1.5),
+    0.5s 0.2s color,
+    0.2s background-color;
+}
+
+.menu-dropdown > *:hover {                  //hover over the dropdown menu
+  background: rgba(0,0,0,0.1);
+}
+
+.menu:not(:hover) > .menu-dropdown > * {   //dropdown menu hidden when you move the cursor
+  visibility: hidden;
+  height: 0;
+  padding-top: 0;
+  padding-bottom: 0;
+  margin: 0;
+  color: rgba(25,25,25,0);
+  transition:
+    0.10s 0.2s height,
+    0.10s 0.2s padding,
+    0.10s 0.2s margin,
+    0.5s color,
+    0.8s visibility;
+  z-index: 99;
+}
+
+.hList {
+  //overflow: hidden;
+  list-style: none; //removing dots from the menu
+  @include web-background; //The @mixin that we can reused throughout the website.
+}
+
+.hList > * {
+  float: left;
+}
+
+.hList > * + * {
```

Comment:
Be careful with using `*` selectors as children like this. It can be problematic.